src/main/java/com/uid2/admin/vertx/service/SiteService.java

```diff
@@ -388,6 +389,9 @@ private void handleSiteUpdate(RoutingContext rc) {
                     ResponseUtil.error(rc, 400, "Invalid parameter for visible: " + visibleParam);
                 }
             }
+            if(name != null) {
```

Comment:
Do we check if the new name is duplicated to the existing sites?

Reply:
Also maybe a sanity check for empty string `name != ''`, we had a bug before where name was set to empty string and caused some hassle debugging :(